pkgs/mime/test/default_extension_map_test.dart

```diff
@@ -2,13 +2,13 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'package:mime/src/default_extension_map.dart';
+import 'package:mime/src/mime_tables.g.dart';
 import 'package:test/test.dart';
 
 void main() {
   group('defaultExtensionMap', () {
     test('keys are lowercase', () {
-      for (final key in defaultExtensionMap.keys) {
+      for (final key in extensionToMime.keys) {
```

Comment:
Consider not changing the name *to* "Mime" instead of "Media", when the original name wasn't called "Mime". It's a free "Media"!